scripts/new-language.md

```diff
@@ -0,0 +1,177 @@
+# Implement a new language
+
+Implementing a new language is _a lot_ of work, so this guide outlines all the necessary steps to do
+so. Ideally these are done sequentially, as separate PRs, that correlate to our tiered language
+support paradigm.
+
+- INIT SCRIPT
+- BIN COMMAND
+- DOCKER PRUNE
+
+## Tier 1
+
+### Add variant to `ProjectLanguage` enum in `moon_config`
+
+This allows projects to configure their primary language, and is utilized by editor extensions.
+
+```rust
+enum ProjectLanguage {
+	// ...
+
+	#[strum(serialize = "kotlin")]
+	Kotlin,
+}
+```
+
+- [ ] Updated enum
+- [ ] Updated TypeScript types at `packages/types/src/project-config.ts`
+- [ ] Verified all `match` callsites handle the new variant
+- [ ] Ran `cargo make json-schemas` and updated the JSON schemas
```

Comment:
Don't think we have cargo make anymore

Reply:
were still using cargo make in the moon repo